clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1258,17 +1391,16 @@ class KernelObjVisitor {
 
   template <typename... HandlerTys>
   void visitFirstArrayElement(const CXXRecordDecl *Owner, FieldDecl *ArrayField,
-                              QualType ElementTy, HandlerTys &... Handlers) {
+                              QualType ElementTy, HandlerTys &...Handlers) {
```

Comment:
Unintentional formatting change

Reply:
Fixed.

Reply:
These format changes keep creeping back. I have once again put the code back the way it was.